src/RoutePluginManager.php

```diff
@@ -24,13 +25,19 @@
  *
  * The manager is marked to not share by default, in order to allow multiple
  * route instances of the same type.
+ *
+ * @see ServiceManager for expected configuration shape
+ *
+ * @template InstanceType of RouteInterface
+ * @extends AbstractPluginManager<InstanceType>
+ * @psalm-import-type ServiceManagerConfiguration from ServiceManager
  */
 class RoutePluginManager extends AbstractPluginManager
 {
     /**
      * Only RouteInterface instances are valid
      *
-     * @var string
+     * @var class-string
```

Comment:
not important, but consider that `interface-string` also exists (AFAIK)

Reply:
Oooh! Didn't know that.